internal/fwserver/server_importresourcestate.go

```diff
@@ -128,6 +143,10 @@ func (s *Server) ImportResourceState(ctx context.Context, req *ImportResourceSta
 		private.Provider = importResp.Private
 	}
 
+	if importResp.DeferredResponse != nil {
+		resp.Deferred = importResp.DeferredResponse
+	}
```

Comment:
super nit: I believe since this is a pointer we could just always set the value (if it's nil, will just continue to be nil)